examples/kms-additional-policy/kms-policy.json

```diff
@@ -0,0 +1,12 @@
+{
+ "Version": "2012-10-17",
+ "Statement": [
+   {
+      "Sid": "AllowAll",
+      "Effect": "Allow",
+      "Principal": "*",
```

Comment:
Nothing functionally wrong, but I wonder if our example should be a wildcard allow.